.env.example

```diff
@@ -0,0 +1 @@
+MONGO_URI=mongodb://localhost:27017/fastify-auth-jwt
```

Comment:
this file is used as reference for all environment variables
do not add real env here, just the reference to be used in the .env file